cloudferrylib/os/network/neutron.py

```diff
@@ -78,17 +78,34 @@ def read_info(self, **kwargs):
         if self.filter_tenant_id is not None:
             shared_nets = self.get_shared_networks_raw()
             for net in shared_nets:
-                LOG.debug("append network ID {}".format(net['id']))
-
                 # do not include the same network twice
                 if net['id'] in [n['id'] for n in nets]:
                     continue
 
+                LOG.debug("Append shared network ID %s", net['id'])
```

Comment:
Yeah, I got it. I mean that it is not obvious for those, who have not worked with CF before... I've already received some questions about such log messages...